python/ray/autoscaler/autoscaler.py

```diff
@@ -496,9 +505,11 @@ def launch_config_ok(self, node_id):
     def files_up_to_date(self, node_id):
         applied = self.provider.node_tags(node_id).get(TAG_RAY_RUNTIME_CONFIG)
         if applied != self.runtime_hash:
-            logger.info(
-                "StandardAutoscaler: {} has runtime state {}, want {}".format(
-                    node_id, applied, self.runtime_hash))
+            logInfo("StandardAutoscaler",
```

Comment:
We can probably remove the "StandardAutoscaler" prefix now that logs include the module in their format string.